examples/siro_sandbox/config/hitl_defaults.yaml

```diff
@@ -0,0 +1,73 @@
+habitat_baselines:
+  # some defaults for interactive HITL applications
+  evaluate: True
+  num_environments: 1
+
+habitat_hitl:
+  window:
+    # title displayed in application title bar GUI
+    title: "Habitat HITL Application"
+    # Horizontal resolution of the window.
```

Comment:
Nearly all of these params have been ported from command-line params, including the help text.